services/ui-src/src/utils/forms/programList.ts

```diff
@@ -0,0 +1,992 @@
+import { FormJson } from "types";
+
+export const generateProgramListFields = (form: FormJson) => {
+  const fields = form.fields;
+
+  const programNameOtherText = {
+    id: "programName-otherText",
+    type: "text",
+    validation: "text",
+    props: { label: "Specify a new program name" },
+  };
+
+  fields.splice(1, 0, programNameOtherText);
+
+  return {
+    ...form,
+    fields: fields,
+  };
+};
+
+// ESP: program list (MCPAR)
+export const programList = {
+  // Alabama
+  AL: [
+    {
+      id: "EIqjAqPHo4tS2geQTeiQrHVz",
+      label: "Alabama Coordinated Health Network (ACHN)",
+    },
+    {
+      id: "24KxJzGtP4NxlUJZrhGJKYEf",
+      label: "Alabama Integrated Care Network (ICN)",
+    },
+    // other, specify
+    {
+      id: "247cNJEIfJ4gbGGrah1e9KPe",
+      label: "Other, specify",
+    },
```

Comment:
🤔 do we need a unique id and other, specify field per report, or could we have one and append it to each just like we do with the otherText field? would there be any issue with them sharing the same id?